README.md

```diff
@@ -13,62 +13,123 @@
+
+![Surprised on Run Failure](screenshots/waifu_app_run_surprise.gif)
+
+### Tasks
+
+Writing code is hard sometimes, your *Waifu* understands.
+So when you trigger various tasks that fail, they will cheer you up when things don't go your way.
+Such as when you build/compile your application, and it fails because you **are missing a darned ;** your *Waifu* has got you covered!
 
-## Waifu Motivation On-demand
-Whenever you feel demotivated on your task, you can request a motivation that randomly plays a sound, and notification to cheer you up. You can invoke it on `Help | Waifu Motivator | Motivate Me` or the shortcut key with (`alt + M` for Windows/Linux and `option + M` for MacOs).
+![Build Failure](screenshots/waifu_build_failure_reaction.gif)
+
+## Personality
+
+Your *Waifu* is a fairly moody state machine and has many reactions to various events such as:
+- She waits for you to come back when you are away for some time
+- She is happy when your test pass.
+- She gets sad when your builds break and tests fail.
+- She gets over come with joy when you fix your tests and make your code compile.
+
+She also has the ability to feel your frustration when **things aren't working, WHY ARE THEY NOT WORKING!!**.
+Your *Waifu* figures its good to inject a little humor into the mix and show you that she is frustrated as well.
+
+![Frustrated Waifu](screenshots/waifu_frustated.gif)
+
+They also have the capability from evolving from being frustrated to full-blown rage.
+This only happens when you have been triggering events in the frustration state for a given period.
+
+Not every person wants their *Waifu* to get frustrated.
+Thankfully, you have the ability to disable this part of her personality, by preventing her from ever being frustrated in the first place! 😄
+
+## Motivation On-demand
+Whenever you feel demotivated on your task, you can request motivation from your *Waifu*.
+Your companion will randomly play a sound and show a *Waifu* to cheer you up.
+You can invoke it on `Help | Waifu Motivator | Motivate Me` or the shortcut key with (`alt + M` for Windows/Linux and `option + M` for MacOs).
 
 <p align="center">
   <img src="screenshots/motivate_me.png" alt="Waifu Motivation On-demand">
 </p>
 
-## Waifu Settings
-Of course, not all the time you're okay with your *Waifu*, there are bad times after all and sometimes you'll get annoyed by them and not wanting to hear anything from them. You can configure it through the settings (`Settings | Other Settings | Waifu Motivator`).
+## Offline Mode
+
+Take your *Waifu* with you as you go.
+You have the ability to write code off the grid and still get motivation events!
+
+## Plugin Settings
+
+Your new *Waifu* is pretty chatty, sometimes that is okay and sometimes it is not.
+Thankfully, you have the ability to configure what your waifu does via the settings menu (`Settings | Other Settings | Waifu Motivator`).
 
 ![Waifu Settings](screenshots/waifu_motivator_settings.png)
```

Comment:
@zd-zero Can I get you to update the settings screen shot and provide all the new screenshots that this new read me has. Please?

Reply:
Yup, I'll work on this. I plan to include the keystrokes displayed on the screen so I'm going to also update the 'Motivation On-demand' section because we also have the visual motivation for that.